README.md

```diff
@@ -101,3 +101,18 @@ We are working on a [markdown version](https://github.com/glpi-project/doc)
  * [TinyMCE v4.7.1](https://github.com/tinymce/tinymce) LGPL2.1
  * [Underscore.js v1.8.3](https://github.com/jashkenas/underscore) MIT
  * [Lodash v unknown](https://github.com/lodash/lodash) MIT
+
+
+## Support
+GLPI is a living software. Improvements are continuously made, new functionality is being developed, and issues are being fixed.
+
+To ease support and development, we need your help when encountering issues.
+There is a GLPI version typical lifecycle:
+ * A new major version (9.3) is released.
+ * Minor versions (9.3.x), fixing bugs or issues, are published approximately every 3 months.
```

Comment:
There is no delay between two bugfixes releases.

Reply:
As writed on the PR comments, it's based on the calendar of past released.
Imho, we should give some visibility to sysadmin about GLPI usual releases :)
Will the addition of a specific note "Durations are given for information only and can be modified" is ok for you ?

Reply:
Maybe talking about "several weeks" between each minor and "several months" between each major?
This should always be true ;)

Reply:
@trasher, please advise my update proposal ?

Regards,
Anael